src/Octoshift/AzureApi.cs

```diff
@@ -15,6 +16,7 @@ public class AzureApi
         private readonly OctoLogger _log;
         private const string CONTAINER_PREFIX = "migration-archives";
         private const int AUTHORIZATION_TIMEOUT_IN_HOURS = 48;
+        private const int DEFAULT_BLOCK_SIZE = 4 * 1024 * 1024;
```

Comment:
I wonder if we should try a lower block size, e.g. 1GB:

> If you have a really good Internet connection, you can go up to 4MB block size. For not so good Internet connections, you can reduce the limit. For example, I always try to use 256-512KB block size as the Internet connection I have is not good.

https://stackoverflow.com/questions/44775415/how-to-choose-blob-block-size-in-azure

Reply:
That could be fun to experiment with... Especially today-- my internet is gonna be pretty 'meh'! 😆

Reply:
Definitely not urgent - we can pick this up later - although I was wondering if it could be related to the performance issues that @hfishback01 mentioned 😊